StripeIdentity/StripeIdentity/Source/NativeComponents/Coordinators/VerificationSheetController.swift

```diff
@@ -239,17 +239,40 @@ final class VerificationSheetController: VerificationSheetControllerProtocol {
         else {
             // Transition to generic error screen
             transitionWithVerificaionPageDataResult(
-                nil,
+                updateDataResult,
                 completion: completion
             )
             return
         }
+
         // If finished collecting, submit and transition
         if updateData.requirements.missing.isEmpty {
-            apiClient.submitIdentityVerificationPage().observe(on: .main) { [weak self] result in
-                self?.isVerificationPageSubmitted = (try? result.get())?.submitted == true
+            apiClient.submitIdentityVerificationPage().observe(on: .main) { [weak self] submittedData in
+                self?.isVerificationPageSubmitted = (try? submittedData.get())?.submittedAndClosed() == true
```

Comment:
nit: since using self? multiple times, usually a good idea to capture self strongly firstl.